modin/conftest.py

```diff
@@ -426,3 +446,8 @@ def TestReadGlobCSVFixture():
     yield
 
     teardown_test_files(filenames)
+
+
+@pytest.fixture
+def get_generated_doc_urls():
+    return lambda: _generated_doc_urls
```

Comment:
Just curious, why do you need `lambda` here instead of just returning `_generated_doc_urls`?

Reply:
Because of how pytest fixtures work - they're computed before a function is executed, and the function I'm using it in is actively trying to load everything to load all the URLs (thus I'm expecting a change _after_ the function finishes).